factfeel_client.py

```diff
@@ -125,11 +125,59 @@ def fact_feel_modify_lights(self, prediction):
 
 class SpeechToText:
     
-    def __init__(self):
+    def __init__(self, device = 2, init = False, debug = False):
```

Comment:
I will put in a hook eventually to store the device index in the config at the UI layer